packages/contracts/scripts/generate-deployed-artifacts.ts

```diff
@@ -50,7 +50,11 @@ const main = async () => {
       )
       artifactNames.push(artifactName)
 
-      content += `const ${artifactName} = require('${relPath}')\n`
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      const artifact = require(relPath)
+      content += `const ${artifactName} = { abi: ${JSON.stringify(
```

Comment:
No longer include the entire artifact, just the address and the ABI. We never use the entire artifact anywhere.